plugins/org.eclipse.sirius.tests.swtbot/src/org/eclipse/sirius/tests/swtbot/propertypage/SiriusSessionDetailsPropertyPageTest.java

```diff
@@ -154,29 +156,31 @@ private String getExpectedSessionDetails() {
         strBuilder.append("*** Resources\n");
```

Comment:
Note that we now use a modern version of Java which supports multi-line strings :smiley:

Reply:
Good point..
I will propose a new patch set